StringCalcTests/StringCalcTest.cs

```diff
@@ -0,0 +1,159 @@
+﻿using StringCalc;
+
+namespace StringCalcTests
+{
+    public class StringCalcTest
+    {
+        private readonly Calculator _calc;
+
+        public StringCalcTest()
+        {
+            // Arrange
+            _calc = new Calculator();
+        }
+
+        [Fact]
+        public void ShouldReturnZeroIfStringIsNull()
+        {
+            // Arrange
+            string? numbers = null;
+
+            // Act
+            var result = _calc.Add(numbers);
+
+            // Assert
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public void ShouldReturnZeroIfStringIsEmpty()
+        {
+            // Arrange
+            var numbers = "";
+
+            // Act
+            var result = _calc.Add(numbers);
+
+            // Assert
+            Assert.Equal(0, result);
```

Comment:
Prefer to have your expected value as a variable in your arrange block as opposed to being a magic number here.